chart/templates/ws-proxy-configmap.yaml

```diff
@@ -40,7 +40,8 @@ data:
                 "connectTimeout": "10s",
                 "idleConnTimeout": "60s",
                 "websocketIdleConnTimeout": "180s",
-                "maxIdleConns": 100
+                "maxIdleConns": 0,
```

Comment:
A comment here would be nice.

Reply:
I added a comment to the commit. Adding it here breaks the build.